Resources/Prototypes/Nyanotrasen/Roles/Jobs/Wildcards/gladiator.yml

```diff
@@ -0,0 +1,25 @@
+- type: job
+  id: Gladiator
```

Comment:
should be setpreference: false

Reply:
done

Reply:
Ok, so that removes it form being able to choose it....How are people supposed to play it?

Reply:
you can still choose it if it's present in the current map proto, which is why we use it for map specific jobs